styleguide/components/31-molecules/fields/checkbox-with-filter/checkbox_filter.functions.js

```diff
@@ -187,9 +187,12 @@
         }
       });
 
-      if (checkboxContainer) {
-        checkboxContainer.style.display = '';
-      }
+      Array.from(checkboxContainers).forEach(container => {
```

Comment:
I think we'll need a polyfill for IE if you want to use this.
Can you check?

Reply:
I assumed babel would've changed it in the compilation process, but apparently it doesn't. I can update the `Array.from`; `Array.prototype.forEach` shouldn't be a problem since it's in the ES5 spec